File: packages/insomnia-httpsnippet/test/fixtures/output/python/python3/full.py
import http.client

conn = http.client.HTTPConnection("mockbin.com")

payload = "foo=bar"

headers = {
    'cookie': "foo=bar; bar=baz",
    'accept': "application/json",
    'content-type': "application/x-www-form-urlencoded"
    }

conn.request("POST", "/har?foo=bar&foo=baz&baz=abc&key=value", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: packages/insomnia-httpsnippet/test/fixtures/output/javascript/fetch/full.js
const fetchOptions = {
  mode: 'cors',
  method: 'POST',
  headers: {
    cookie: 'foo=bar; bar=baz',
    accept: 'application/json',
    'content-type': 'application/x-www-form-urlencoded',
  },
  body: {
    foo: 'bar',
  },
};

fetch('http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value', fetchOptions)
  .then(response => response.json())
  .then(data => console.log(data));
